cms/migrations/0002_auto_20200505_0933.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.10 on 2020-05-05 09:33
+
+from django.db import migrations, models
+import django.db.models.deletion
+import wagtail.core.blocks
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0045_assign_unlock_grouppagepermission'),
+        ('wagtailforms', '0004_add_verbose_name_plural'),
+        ('klasses', '0014_no_max_len_application_stage'),
+        ('wagtailimages', '0001_squashed_0021'),
+        ('wagtailredirects', '0006_redirect_increase_max_length'),
+        ('cms', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BootCampRunPage',
```

Comment:
Please rename to `BootcampRunPage` (lowercase "c") following the convention from production website.